packages/storybook8/stories/StatefulClient/ReactHooks/UseSelector.mdx

```diff
@@ -0,0 +1,154 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Stateful Client/React Hooks/UseSelector" />
```

Comment:
```suggestion
<Meta title="Stateful Client/React Hooks/useSelector" />
```

Same comment here

Reply:
In current storybook we have these hooks starting with capital, keeping it same as of now, will update later if feel we should switch 
<img width="227" alt="Screenshot 2024-07-26 at 10 10 25 AM" src="https://github.com/user-attachments/assets/83642e21-1d08-410c-a3e4-ae2453e65034">